.env.example

```diff
@@ -54,6 +54,10 @@ FILE_STORE_PROVIDER='local'
 # Google Cloud credentials are required to use FILE_STORE_PROVIDER='gcs'
 # GOOGLE_GCS_BUCKET='BUCKET_NAME'
 
+# WEB SERVER
+# Time window, in seconds, used by the web server to kick its users off. If empty, it has a default value.
```

Comment:
+1 Maybe mention the default value? "If empty it defaults to 60s."

Reply:
Yeah, I wanted to avoid that, just in case we changed the default value one day. That said, minor change and clearer information so I'm adding that